src/Imdb/Title.php

```diff
@@ -2614,15 +2614,13 @@ public function releaseInfo()
     public function locations()
     {
         if (empty($this->locations)) {
-            $page = $this->getPage("Locations");
-            if (empty($page)) {
+            $xpath = $this->getXpathPage("Locations");
+            if (empty($xpath)) {
                 return array();
             } // no such page
-            $doc = new \DOMDocument();
-            @$doc->loadHTML($page);
-            $xp = new \DOMXPath($doc);
-            $cells = $xp->query("//section[@id=\"filming_locations\"]//dt");
+            $cells = $xpath->query("//section[@id=\"filming_locations\"]//dt");
             foreach ($cells as $cell) {
+                $dt = $xpath->query($cell->getNodePath() . '//dt')->item(0)->nodeValue;
```

Comment:
It seems travis are crashing on this one. Can't see it beeing used, so should revert that one as well I'm afraid.

Reply:
...oeps left over i guess